easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.12-iimkl-2023b.eb

```diff
@@ -0,0 +1,116 @@
+            {'numpy-1.25.1_disable-broken-fortran-docstring-test.patch':
+             '8a4d36e3b3a9c9bf43df6e5214f3883234a069b80c5c1027a7c84bd5cb133457'},
+            {'numpy-1.25.1_fix-test_features.patch':
+             '1c05ee5d105fe2f824416dd6dd5c64ed0c1cd710a002b4e6dbfafff19203adc5'},
+        ],
+    }),
+    ('ply', '3.11', {
+        'checksums': ['00c7c1aaa88358b9c765b6d3000c6eec0ba42abca5351b095321aef446081da3'],
+    }),
+    ('gast', '0.5.4', {
+        'checksums': ['9c270fe5f4b130969b54174de7db4e764b09b4f7f67ccfc32480e29f78348d97'],
+    }),
+    ('beniget', '0.4.1', {
+        'checksums': ['75554b3b8ad0553ce2f607627dad3d95c60c441189875b98e097528f8e23ac0c'],
+    }),
+    ('pythran', '0.14.0', {
+        'checksums': ['42f3473946205964844eff7f750e2541afb2006d53475d708f5ff2d048db89bd'],
+    }),
+    ('versioneer', '0.29', {
+        'checksums': ['5ab283b9857211d61b53318b7c792cf68e798e765ee17c27ade9f6c924235731'],
+    }),
+    ('scipy', '1.12.0', {
+        'enable_slow_tests': True,
+        'ignore_test_result': False,
+        'patches': [
+            'scipy-1.11.1_disable-tests.patch',
+            'scipy-1.11.1_xfail-aarch64_test_maxiter_worsening.patch',
+            'scipy-1.11.4_disable-test_branch_cut.patch',  # intel might need -assume=minus0
+            'scipy-1.12.0_disable-tests-iimkl.patch',
+        ],
+        'checksums': [
+            {'scipy-1.12.0.tar.gz': '4bf5abab8a36d20193c698b0f1fc282c1d083c94723902c447e5d2f1780936a3'},
+            {'scipy-1.11.1_disable-tests.patch': '906bfb03397d94882ccdc1b93bc2c8e854e0e060c2d107c83042992394e6a4af'},
+            {'scipy-1.11.1_xfail-aarch64_test_maxiter_worsening.patch':
+             '918c8e6fa8215d459126f267764c961bde729ea4a116c7f6287cddfdc58ffcea'},
+            {'scipy-1.11.4_disable-test_branch_cut.patch':
+             '3433d284e4074ee8a5b10bfc65c729214bc75670c36d8b7a983bb5ccddc5cf61'},
+            {'scipy-1.12.0_disable-tests-iimkl.patch':
+             'edf575be62b77bb8984e87e75f69a0799d719fc8b5c801c79e4f9592e1c8203f'},
+        ],
+    }),
+    ('numexpr', '2.8.7', {
+        'checksums': ['596eeb3bbfebc912f4b6eaaf842b61ba722cebdb8bc42dfefa657d3a74953849'],
+    }),
+    ('Bottleneck', '1.3.7', {
+        'checksums': ['e1467e373ad469da340ed0ff283214d6531cc08bfdca2083361a3aa6470681f8'],
+    }),
+    ('tzdata', '2023.3', {
+        'checksums': ['11ef1e08e54acb0d4f95bdb1be05da659673de4acbd21bf9c69e94cc5e907a3a'],
+    }),
+    ('pandas', '2.1.3', {
+        'preinstallopts': "export PANDAS_CI=0 && ",
+        'checksums': ['22929f84bca106921917eb73c1521317ddd0a4c71b395bcf767a106e3494209f'],
+    }),
+    ('mpmath', '1.3.0', {
+        'checksums': ['7a28eb2a9774d00c7bc92411c19a89209d5da7c4c9a9e227be8330a23a25b91f'],
+    }),
+    ('deap', '1.4.1', {
+        'modulename': 'deap.base',
+        'checksums': ['cc01de9892dfa7d1bc9803dab28892fead177f0182c81db47360a240ead778ff'],
+    }),
+]
+
+sanity_pip_check = True
```

Comment:
This line also needs removing:
```suggestion
```

Reply:
I removed the line because the test suite doesn't like it.